src/commcare_cloud/ansible/roles/elasticsearch/defaults/main.yml

```diff
@@ -17,3 +17,5 @@ backup_es_swift: no
 elasticsearch_enable_inline_groovy_scripts: True
 elasticsearch_service_name: elasticsearch
 elasticsearch_node_name: "{{ hostname }}"
+elasticsearch_deletelogs: False
+delete_older_logs_days: 7
```

Comment:
All ansible variable names are global, so this has to start with elasticsearch to make it clear and not conflict. So maybe something like `elasticsearch_log_retention_days: 7`. Also I'd rename `elasticsearch_deletelogs` to `elasticsearch_delete_old_logs`